doc/apiary/apiary.apib

```diff
@@ -119,6 +119,50 @@ Statistics can be reset by a PUT request.
 # Group Grouping Rules API
 API operations used to manage the grouping rules of Cygnus.
 
+## Methods [/Name Mappings API]
+
+### "GET /v1/namemappings" [GET /v1/namemappings]
+Name mappings can be retrieved by GET methods.
+
++ Request 
+
+      GET http://<cygnus_host>:<management_port>/v1/namemappings
+
++ Response 200 (application/json)
+
+        {
+            "success":"true",
+            "serviceMapping": [
+                {
+                    "originalService":"",
+                    "newService":"",
+                    "servicePathMappings": [
+                        {
+                            "originalServicePath":"",
+                            "newServicePath":"",
+                            "entityMappings": [
+                                {
+                                    "originalEntityId":"",
+                                    "originalEntityType":"",
+                                    "newEntityId":"",
+                                    "newEntityType":"",
+                                    "attributeMappings":[
+                                        {
+                                            "originalAttributeName":"",
+                                            "originalAttributeType":"",
+                                            "newAttributeName":"null",
```

Comment:
In the .apib examples avoid using "null" or empty values (i.e. try writting something typical such as "Room1" of type "Room" :))